src/controller/CHIPDeviceController.h

```diff
@@ -336,9 +336,11 @@ class DLL_EXPORT DeviceController : public Messaging::ExchangeDelegate,
 
     void PersistNextKeyId();
 
-    FabricIndex mFabricIndex = 0;
+    FabricIndex mFabricIndex = 1;
     Transport::FabricTable mFabrics;
 
+    Crypto::P256Keypair mOperationalKey;
```

Comment:
This keypair is only needed during CASE. It does not need to actually be maintained in this context. I guess this can be a refactor for later, but it's an important consideration: Nodes do not need access to credentials at all times, only during single message processing steps of the session establishment. The rest of the time, all necessary data is:

- Basic fabric metadata for all fabrics:
  - Root public key associated with the fabric
  - Fabric ID
  - Node ID
- PASE/CASE/Group session contexts that actually contain session-specific keys in use

Reply:
This will actually get removed once we move the commissioner's credentials generation out of this class.